ubuntudesign/www.ubuntu.com
# Remove autopilot testdrive

## Done

* We are getting rid of the Autopilot testdrive, this PR removes it from the site
 * Removed references to the testdrive from cloud section
 * Simplified /download/cloud to have the single page with instructions
 * Removed older instructions pages and created redirects
 * Updated other references and links to instructions

## QA

[demo site](http://www.ubuntu.com-remove-autopilot-testdrive.demo.haus/)

1. go to /download/cloud [copy doc](https://docs.google.com/document/d/1y3uaLNn7dxZKcLq2OmVJ96o3NOqn4OTJflty2GmxV9I/edit) and see that it works and matches the copy doc
2. go to /cloud [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit)  and see that it works and matches the copy doc
3. go to /cloud/openstack [copy doc](https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit) and see that it works and matches the copy doc
4. go to /cloud/openstack/autopilot [copy doc](https://docs.google.com/document/d/1VmFtp_gEYvioRgGXbiQZpVGBhlJoIZAirLk6IOVUjUE/edit) and see that it works and matches the copy doc
5. see that the ebook download on /download/cloud still works
6. see that the redirects work (download/cloud/install-autopilot-testdrive/ > /download/cloud & download/cloud/install-openstack-with-autopilot/ > /download/cloud)

## Issue / Card

[trello card](https://trello.com/c/9MugT9N6)
